cmd/juju/application/deployer/bundlehandler.go

```diff
@@ -507,9 +457,7 @@ func (h *bundleHandler) constructChannelAndOrigin(curl *charm.URL, charmBase ser
 
 func (h *bundleHandler) getChanges() error {
 	bundleURL := ""
-	if h.bundleURL != nil {
-		bundleURL = h.bundleURL.String()
-	}
```

Comment:
I'm wondering about this change. The comments on the structure element may be misleading.

Have you run the deploy bundle integration tests against this change?

Reply:
Running the tests right now, so far so good: 

```
===> [ ✔ ] Success: deploy bundle (225s)
===> [ ✔ ] Success: deploy bundle overlay (125s)
===> [   ] Running: deploy exported charmhub bundle with fixed revisions
```

I'll update this when the test completes.

Reply:
Yup, you were right about this, bringing it back :+1: